packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2003,9 +2003,19 @@ export class ContainerRuntime
 						initialDelayMs: this.initialSummarizerDelayMs,
 					},
 				);
-				this.summaryManager.on("summarize", (eventProps) => {
-					this.emit("summarize", eventProps);
+				// Forward events from SummaryManager
+				[
+					"summarize",
+					"summarizeAllAttemptsFailed",
+					"summarizerStop",
+					"summarizerStart",
+					"summarizerStartupFailed",
+				].forEach((eventName) => {
+					this.summaryManager?.on(eventName, (...args: any[]) => {
+						this.emit(eventName, ...args);
```

Comment:
you emit these, but they are not exposed in the container runtime typing at all. I also wonder what the expected usage of these events is, and if each really needs it own event, or if we could have a single event, but a discriminated union of property objects. if the goal is logging, i think a combined even is easier to managed.

consider adding a scenario test under packages\test\local-server-tests to illustrate usage.

Reply:
This is a good point. I'll move the typing to be off the `IContainerRuntime` interface instead.
These events are to help consumers (i.e. Loop) better observe when the summarizer is failing.